python/artm/batches_utils.py

```diff
@@ -223,6 +223,11 @@ def __reset_batch():
             batch.id = str(uuid.uuid4())
```

Comment:
This change should be covered by new test in this file

https://github.com/bigartm/bigartm/blob/master/python/tests/artm/test_batches_utils.py